src/main/g8/build.sbt

```diff
@@ -12,7 +11,6 @@ lazy val root = (project in file(".")).
       "com.typesafe.akka" %% "akka-http"            % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-xml"        % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-stream"          % akkaVersion,
```

Comment:
reverted, the concern is with
```
[warn] There may be incompatibilities among your library dependencies.
[warn] Here are some of the libraries that were evicted:
[warn]     * com.typesafe.akka:akka-stream_2.12:2.4.19 -> 2.5.3
[warn] Run 'evicted' to see detailed eviction warnings
```
setting akka_version=2.4.19 in default.properties would get rid of the warning